src/verifact_manager.py

```
@@ -128,15 +132,15 @@
         result = await Runner.run(verdict_writer_agent, prompt)
         return result.final_output_as(Verdict)
 
-    async def _generate_all_verdicts(self, claims_with_evidence: list[tuple[Claim, list[Evidence]]]) -> list[Verdict]:
+    async def _generate_all_verdicts(self, claims_with_evidence: List[tuple[Claim, Optional[List[Evidence]]]]) -> List[Verdict]:
```

Ruff (0.11.9)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)